apps/dashboard/app/views/files/_files_table.html.erb

```diff
@@ -24,6 +24,13 @@
       <tbody>
       </tbody>
     </table>
+
+    <div class="text-center text-primary" style="display:none" id="tloading-spinner">
```

Comment:
Instead of inline styles, we should be using the class `d-none`. inline styles are a CSP violation (or at least will be at some point) so we should avoid using them.

So I think instead of the jQuery API for `hide` & `show`, we should just edit the classList in plain javascript.

```javascript
// API to add to remove from the classList
document.getElementById('directory-contents').classList.remove('d-none');

// API to add to the classList
document.getElementById('directory-contents').classList.add('d-none');
```